lib/job.js

```diff
@@ -10,6 +10,7 @@ const tokenGen = Symbol('tokenGen');
 const apiUri = Symbol('apiUri');
 const START_INDEX = 3;
 const MAX_COUNT = 1000;
```

Comment:
This can also be more detailed name :pray:

Reply:
I changed.